database/mysql.js

```diff
@@ -1,28 +1,31 @@
 const mysql = require('mysql');
-const featuresTable = 'CREATE TABLE IF NOT EXISTS features (house_id int NOT NULL AUTO_INCREMENT, type varchar(255), year_built varchar (255), heating varchar (255), cooling varchar (255), parking varchar (255),lot varchar (255), days_on_zillow varchar (255), price_per_sqft varchar (255), PRIMARY KEY (house_id));'
-const interiorTable = 'CREATE TABLE IF NOT EXISTS interior_features (feature_id int NOT NULL AUTO_INCREMENT, bedrooms varchar(255), bathrooms varchar (255), interiorheating varchar (255), interiorcooling varchar (255), appliances varchar (255), kitchen varchar (255), flooring varchar (255), house_id varchar (255), sqft varchar (255), PRIMARY KEY (feature_id));'
+const featuresTable = 'CREATE TABLE IF NOT EXISTS features (house_id int NOT NULL AUTO_INCREMENT, type varchar(255), year_built int, heating varchar (255), cooling varchar (255), parking varchar (255),lot int, days_on_zillow int, bedrooms float, bathrooms float, interiorheating varchar (255), interiorcooling varchar (255), appliances varchar (255), kitchen varchar (255), flooring varchar (255), sqft int, PRIMARY KEY (house_id));'
 const dbName = 'zillow';
 const mockFeatures = require('./mockFeatures');
 const mockInterior = require('./mockInterior');
 const features = require('./createMockData');

```

Comment:
This is a nit pick suggestion, but may help readability. On Line 7 since you are only importing the createFeatures function from createMockData.js why not specific the function in your require statement. Here is an example:

`const createFeatures = require('./createMockData').createFeatures;`

The other easy fix would be to just export the function instead of putting it in an object on line 73 of  your createMockData file. Here's what I mean:

`module.exports = createFeatures;`

Reply:
True, left over from my refactor where I deleted the interiorFeatures function.